portal-frontend/src/app/features/applications/edit-submission/primary-contact/primary-contact.component.html

```diff
@@ -161,7 +161,7 @@ <h6>Authorization Letters (if applicable)</h6>
       (deleteFile)="onDeleteFile($event)"
       (openFile)="openFile($event)"
       [showErrors]="showErrors"
-      [isRequired]="needsAuthorizationLetter"
+      [isRequired]="needsAuthorizationLetter || hasCrownParcels"
```

Comment:
question: I feel like all conditions for needing an authorization letter should be captured by `needsAuthorizationLetter`. Can we move `hasCrownParcels` into there?

Reply:
Great idea!